pkg/kubelet/apis/config/types.go

```diff
@@ -656,7 +656,8 @@ type ShutdownGracePeriodByPodPriority struct {
 
 type MemorySwapConfiguration struct {
 	// swapBehavior configures swap memory available to container workloads. May be one of
-	// "", "LimitedSwap": workload combined memory and swap usage cannot exceed pod memory limit
+	// "", "NoSwap": workloads can not use swap, default option.
+	// "LimitedSwap": workload combined memory and swap usage cannot exceed pod memory limit
```

Comment:
Although it's not directly related to your changes, this documentation seems to be inaccurate.
Perhaps something as follows would be better:

```suggestion
	// "LimitedSwap": workload swap usage is limited. The swap limit is proportionate to the container's memory request.
```

The limit formula (as can be seen [here](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2400-node-swap/README.md#steps-to-calculate-swap-limit)) is:
`limit=<swap_size>*(<memory_request>/<node_memory_capacity>)`.

I can take care of this (and maybe other instances of outdated documentation?) in a separate PR if you think it's out-of-scope for this one.

Reply:
Good call. I'll add this.

Reply:
updated.